pkg/operator/target_config_reconciler.go

```diff
@@ -158,6 +163,11 @@ func (c *TargetConfigReconciler) manageConfigMap(secondaryScheduler *secondarysc
 	return resourceapply.ApplyConfigMap(c.kubeClient.CoreV1(), c.eventRecorder, required)
 }
 
+func (c *TargetConfigReconciler) getConfigMapResourceVersion(secondaryScheduler *secondaryschedulersv1.SecondaryScheduler) string {
+	required := resourceread.ReadServiceAccountV1OrDie(bindata.MustAsset("assets/secondary-scheduler/configmap.yaml"))
+	return required.ObjectMeta.ResourceVersion
```

Comment:
`required.ObjectMeta.ResourceVersion` will always be the same. The code needs to read the actual configmap (using an informer) and return its `ResourceVersion` instead.